bc/channel/utils/connectors/twitter.py

```diff
@@ -48,12 +72,19 @@ def add_status(
         media_array = []
         payload: dict[str, Any] = {"text": message}
         if text_image:
-            media_response = self.api.request(
-                "media/upload", None, {"media": text_image.to_bytes()}
+            media_id = self.upload_media(
+                text_image.to_bytes(),
+                f"An image of the entry's full text: {text_image.description}",
             )
-            media_id = media_response.json()["media_id"]
             media_array.append(str(media_id))
+            payload["media"] = {"media_ids": media_array}
 
+        if thumbnails:
+            for idx, thumbnail in enumerate(thumbnails):
+                media_id = self.upload_media(
+                    thumbnail, f"Thumbnail of page {idx + 1}"
```

Comment:
Here and above, it'd be a bit better to say:

```suggestion
                    thumbnail, f"Thumbnail of page {idx + 1} of the PDF"
```

Reply:
got it! I will update the `alt text` for the thumbnails